application/migrations/005_write_keywords_initial_cache.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
We don't actually need this migration, do? The cron job will do the same thing, we just have to make sure to turn it on, and ideally to run it manually once after deploying this code in production...